.pipelines/pipeline.user.windows.yml

```diff
@@ -0,0 +1,61 @@
+environment:
+  host:
+    os: 'windows'
+    flavor: 'server'
+    version: '2019'
+  runtime:
+    provider: 'appcontainer'
+    image: 'cdpxwinrs5.azurecr.io/global/vse2019/16.3.7:latest'
```

Comment:
>cdpxwinrs5.azurecr.io/global/vse2019/16.3.7:latest [](start = 12, length = 50)

@ivadim Is this an ok image for us to use? (We don't need to be as cutting edge as the ASP.NET Core 3.0 repo).